schema/annotation-source.yaml

```diff
@@ -185,7 +190,23 @@ $defs:
         enum: [ "supports", "uncertain", "opposes" ]
         description: >-
           The direction of this statement with respect to the target proposition.
-
+      has_evidence_of_type: 
```

Comment:
ClinVar collection method data is currently in a "clinvar-scv-methods" Extension, but could be framed to fit into the VA `Statement.has_evidence_of_type` attribute.